nixos/modules/security/wrappers/wrapper.nix

```
@@ -0,0 +1,21 @@
+{ stdenv, linuxHeaders, parentWrapperDir, debug ? false }:
+# For testing:
+# $ nix-build -E 'with import <nixpkgs> {}; pkgs.callPackage ./wrapper.nix { parentWrapperDir = "/run/wrappers"; debug = true; }'
+stdenv.mkDerivation {
+  name = "security-wrapper";
+  buildInputs = [ linuxHeaders ];
+  dontUnpack = true;
+  hardeningEnable = [ "pie" ];
+  CFLAGS = [
+    "-DWRAPPER_DIR=\"${parentWrapperDir}\""
```

Any reason to not do:

```suggestion
    ''-DWRAPPER_DIR="${parentWrapperDir}"''
```

?